src/utils/fs.rs

```diff
@@ -44,10 +45,20 @@ pub fn get_all_files(paths: &[&str],
     walker.build_parallel().run(move|| {
         let tx = tx.clone();
         Box::new(move |entry| {
-
             let entry = match entry {
                 Ok(entry) => entry,
                 Err(error) => {
+                    use ignore::Error;
+                    if let Error::WithDepth { err: ref error, .. } = error {
+                        if let Error::WithPath { ref path, err: ref error } = **error {
+                            if let Error::Io(ref error) = **error {
```

Comment:
I think we can condense this code. Instead of just checking for Permission denied, couldn't we have `error!("{} reading {}", error.description(), path.display());`?

Reply:
Yep, this is indeed better, I guess I was too focused on the PermissionDenied error there. Looks a lot better like this and makes more sense overall